docs/sources/reference/components/loki/loki.source.journal.md

```diff
@@ -33,37 +33,86 @@ loki.source.journal "<LABEL>" {
+| `max_age`        | `duration`           | The oldest relative time from process start that {{< param "PRODUCT_NAME" >}} reads. | `"7h"`  | no       |
+| `path`           | `string`             | Path to a directory to read entries from.                                            | `""`    | no       |
+| `relabel_rules`  | `RelabelRules`       | Relabeling rules to apply on log entries.                                            | `{}`    | no       |
 
 {{< admonition type="note" >}}
-A `job` label is added with the full name of the component `loki.source.journal.LABEL`.
+{{< param "FULL_PRODUCT_NAME" >}} adds a `job` label with the full name of the component `loki.source.journal.LABEL`.
 {{< /admonition >}}
 
-When the `format_as_json` argument is true, log messages are passed through as JSON with all of the original fields from the journal entry.
-Otherwise, the log message is taken from the content of the `MESSAGE` field from the journal entry.
+When the `format_as_json` argument is true, log messages pass through as JSON with all of the original fields from the journal entry.
+Otherwise, {{< param "PRODUCT_NAME" >}} takes the log message from the content of the `MESSAGE` field from the journal entry.
 
-When the `path` argument is empty, `/var/log/journal` and `/run/log/journal` are used for discovering journal entries.
+When the `path` argument is empty, {{< param "PRODUCT_NAME" >}} uses `/var/log/journal` and `/run/log/journal` for discovering journal entries.
 
 The `relabel_rules` argument can make use of the `rules` export value from a [`loki.relabel`][loki.relabel] component to apply one or more relabeling rules to log entries before they're forwarded to the list of receivers in `forward_to`.
 
-All messages read from the journal include internal labels following the pattern of `__journal_FIELDNAME` and are dropped before sending to the list of receivers specified in `forward_to`.
-To keep these labels, use the `relabel_rules` argument and relabel them to not be prefixed with `__`.
+All messages read from the journal include internal labels following the pattern of `__journal_FIELDNAME` and {{< param "PRODUCT_NAME" >}} drops them before sending to the list of receivers specified in `forward_to`.
+To keep these labels, use the `relabel_rules` argument and relabel them to not have the `__` prefix.
 
 {{< admonition type="note" >}}
 Many field names from journald start with an `_`, such as `_systemd_unit`.
 The final internal label name would be `__journal__systemd_unit`, with _two_ underscores between `__journal` and `systemd_unit`.
+
+Additionally, the `PRIORITY` field receives special handling and creates two labels:
+
+* `__journal_priority` - The numeric priority value between 0 and 7
+* `__journal_priority_keyword` - The priority keyword, for example `emerg`, `alert`, `crit`, `error`, `warning`, `notice`, `info`, or `debug`
```

Comment:
Based on the comment above, shouldn't those be `__journal__priority` and `__journal__priority_keyword`?